### PR TITLE
feat(registration): login with unfinished registration

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**
## Description

Login flow when registration is unfinished

## Why

Registration started but not completed and if user try to login in portal then redirect accordingly

## Issue

n/a

## Checklist

Please delete options that are not relevant.

- [X] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [X] I have performed [IP checks](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool) for added or updated 3rd party libraries
- [X] I have performed a self-review of my own code
- [X] I have successfully tested my changes locally

### CODE REVIEW

**Review by oyo — APPROVED** [review]

File: src/components/pages/Registration/index.tsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+      companyData?.applicationStatus === ApplicationStatus.SELECT_COMPANY_ROLE
+    ) {
+      setActiveStep(3)
+    } else if (
+      companyData?.applicationStatus === ApplicationStatus.UPLOAD_DOCUMENTS
+    ) {
+      setActiveStep(4)
+    } else if (companyData?.applicationStatus === ApplicationStatus.VERIFY) {
+      setActiveStep(5)
+    }
```
Comment (oyo) [inline src/components/pages/Registration/index.tsx:58]: please use a switch statement instead of this long chain of if-else

**Review by oyo — CHANGES_REQUESTED** [review]
if-else to switch

File: src/assets/locales/de/main.json [hunk 2]
```diff
@@ -104,6 +104,22 @@
...
+        "step1": "Verify your company data",
+        "step2": "Responsibilities & admin account",
+        "step3": "Role of your company",
+        "step4": "Upload IDs and certificates",
+        "step5": "Verify your entries",
+        "tooltipText": "Continue where you left"
+      },
+      "helpText": "If you have any questions regarding your registration please contact our Helpdesk:",
+      "email": "help@catena-x.com"
+    },
```
Comment (jjeroch) [inline src/assets/locales/de/main.json:122]: @nidhigarg-bmw please transfer the messages to german

**Review by jjeroch — CHANGES_REQUESTED** [review]

File: src/assets/locales/de/main.json (hunk 2 above)
Comment (nidhigarg-bmw) [reply src/assets/locales/de/main.json:122]: @jjeroch Made the translations

File: src/components/pages/Registration/index.tsx (hunk 1 above)
Comment (nidhigarg-bmw) [reply src/components/pages/Registration/index.tsx:55]: @oyo Made the change. Please review now

**Review by oyo — APPROVED** [review]

**Pull request merged**